frontend/src/components/common/CourseDetails.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export const CourseDetails = ({ data = {} }) => {
+  const {
+    is_closed: isClosed,
+    is_cancelled: isCanceled,
+    section_id_normalized: sectionId,
+    meetings,
+  } = data
+  const isOpen = !isClosed && !isCanceled
+  const meetingDates = meetings.map(
+    ({ meeting_days: days, start_time: start, end_time: end }) =>
+      `${days} ${start} - ${end}`
+  )
+  return (
+    <li>
+      {sectionId}
+      <i className={`ml-2 fa fa-fw fa-${isOpen ? 'check' : 'times'}`} />
+      <span className="ml-2" style={{ color: '#aaa' }}>
```

Comment:
shared constants for colors would be good (that can be at a later PR)

Reply:
added a ticket in clubhouse